app/views/hackathon/digest_mailer/digest.text.erb

```diff
@@ -0,0 +1,5 @@
+Check out these <%= @digest.listings.count %> hackathons coming up near you.
```

Comment:
```suggestion
Check out these hackathons coming up near you!
```

If there's one it'd be plural and the complexity just doesn't justify a number at the top.

Reply:
We decided to keep it since it makes the email feel more personalized

Fixed up the pluralization: [`dd038c6` (#76)](https://github.com/hackclub/hackathons-backend/pull/76/commits/dd038c6045d642dda638227468b8d82b2dd6cf2b)